docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to [hear your feedback](mailto: product@goteleport.com).
+We currently do not recommend using this feature in production.
+</Admonition>
+
+Teleport's Workload Identity issues flexible short-lived identities intended
```

Comment:
I would add 1-2 general sentences about what short-lived identifies for workloads entail. What is a workload in this context? How does this differ from regular Machine ID?